concent_api/core/validation.py

```diff
@@ -132,3 +136,26 @@ def validate_golem_message_signed_with_key(
                 public_key
             )
         )
+    if isinstance(golem_message, message.tasks.TaskToCompute):
```

Comment:
Why is this here ? It is not related to what this function does. There is a function for validating TTC, please move it there.

Reply:
Done